coverart_redirect/request.py

```diff
@@ -244,6 +244,10 @@ def handle_robots(self):
         '''Serve up a permissive robots.txt'''
         return Response(response="User-agent: *\nAllow: /", mimetype='text/plain')
 
+    def handle_api(self):
+        '''Redirect to API docs at musicbrainz.org'''
+        return request.redirect (code=301, location='https://musicbrainz.org/doc/Cover_Art_Archive/API')
```

Comment:
Oops. A space too much, eh? Anyway, I don't want to bother pulling this out again and fixing it, if it's not something we want.